lib/harfbuzz.rb

```diff
@@ -6,7 +6,18 @@ module Harfbuzz
 
   extend FFI::Library
 
-  ffi_lib 'harfbuzz'
+  lib_names = ['harfbuzz']
+
+  # Windows-specific logic
+  if RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/
+    lib_names.unshift('libharfbuzz-0')
+
+    if defined?(RubyInstaller::Runtime)
+      RubyInstaller::Runtime.add_dll_directory("#{RbConfig::CONFIG['prefix']}/msys64/mingw64/bin")
+    end
```

Comment:
@luislavena please let me know if there is a better way to do this?

I looked at your guide here: https://github.com/oneclick/rubyinstaller2/wiki/For-gem-developers#-dll-loading and I couldn't figure it out. Also wondering why RubyInstaller RIDK doesn't add `<ruby dir>/msys64/mingw64/bin` by default?

Reply:
Sorry stupid question; I needed to install the UCRT version 🤦